src/GraphQL.SystemReactive/SubscriptionExecutionStrategy.cs

```diff
@@ -50,45 +47,19 @@ public override async Task<ExecutionResult> ExecuteAsync(ExecutionContext contex
         {
             context.CancellationToken.ThrowIfCancellationRequested();
 
-            var arguments = ExecutionHelper.GetArguments(
-                node.FieldDefinition!.Arguments,
-                node.Field!.Arguments,
-                context.Variables);
-
             object? source = (node.Parent != null)
                 ? node.Parent.Result
                 : context.RootValue;
 
             try
             {
-                var resolveContext = new ResolveEventStreamContext
-                {
-                    FieldAst = node.Field,
-                    FieldDefinition = node.FieldDefinition,
-                    ParentType = node.GetParentType(context.Schema)!,
-                    Arguments = arguments,
-                    Source = source,
-                    Schema = context.Schema,
-                    Document = context.Document,
-                    RootValue = context.RootValue,
-                    UserContext = context.UserContext,
-                    Operation = context.Operation,
-                    Variables = context.Variables,
-                    CancellationToken = context.CancellationToken,
-                    Metrics = context.Metrics,
-                    Errors = context.Errors,
-                    Path = node.Path,
-                    RequestServices = context.RequestServices,
-                };
-
-                var eventStreamField = node.FieldDefinition as EventStreamFieldType;
-
+                var resolveContext = new ResolveFieldContext(new ReadonlyResolveFieldContext(node, context));
```

Comment:
Wrapped because of ` resolveContext.RequestServices = null;` ?

Reply:
yes.